doc/man3/SSL_CTX_set_cookie_generate_cb.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_cookie_generate_cb,
+SSL_CTX_set_cookie_verify_cb
+- Callback functions for DTLS and statless TLS1.3 cookies
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void SSL_CTX_set_cookie_generate_cb(SSL_CTX *ctx,
+                                     int (*app_gen_cookie_cb) (SSL *ssl,
+                                                               unsigned char
+                                                               *cookie,
+                                                               unsigned int
+                                                               *cookie_len));
+ void SSL_CTX_set_cookie_verify_cb(SSL_CTX *ctx,
+                                   int (*app_verify_cookie_cb) (SSL *ssl,
+                                                                const unsigned char
+                                                                *cookie,
+                                                                unsigned int
+                                                                cookie_len));
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_cookie_generate_cb() sets the callback used by L<DTLSv1_listen(3)>
+and L<SSL_stateless(3)> to generate the application-controlled portion of the
+cookie provided to clients in the HelloRetryRequest transmitted as a response to
+a ClientHello with a missing or invalid cookie. app_gen_cookie_cb() must write
+at most SSL_COOKIE_LENGTH bytes into B<cookie>, and must write the number of
+bytes written to B<cookie_len>. If a cookie cannot be generated, a zero return
+value can be used to abort the handshake.
+
+SSL_CTX_set_cookie_verify_cb() sets the callback used by L<DTLSv1_listen(3)> and
+L<SSL_stateless(3)> to determine whether the application-controlled portion of a
+ClientHello cookie is valid. A nonzero return value from app_verify_cookie_cb()
+communicates that the cookie is valid. The integrity of the entire cookie,
+including the application-controlled portion, is automatically verified using a
+HMAC before app_verify_cookie_cb() is called.
```

Comment:
The HMAC thing only applies to SSL_stateless().

(As an aside I do wonder whether it would be wise to separate the DTLSv1_listen() cookie callback from the SSL_stateless() cookie callback - although superficially similar they are used differently which is kind of confusing).